src/components/ga4-opl/template.njk

```diff
@@ -0,0 +1,17 @@
+<script {% if params.nonce%} nonce='{{ params.nonce }}'{%  endif %}>
+    window.addEventListener('load', function () {
+        window
+            .DI
```

Comment:
do you think we should add the window.DI?.appInit , it was suggested when I implemented it in the Home repo , just in case it is undefined , it doesn't throw an error ?

Reply:
@di-aholme don't think this has been added/changed yet